examples/dcgan_fashiongen/dcgan_fashiongen_handler.py

```diff
@@ -0,0 +1,86 @@
+import os
+import zipfile
+import torch
+from io import BytesIO
+from torchvision.utils import save_image
+from ts.torch_handler.base_handler import BaseHandler
+
+MODELSZIP = "models.zip"
+CHECKPOINT = "DCGAN_fashionGen-1d67302.pth"
```

Comment:
Can we rename this to a generic name when we wget it in the `create_mar.sh` script so that we won't have to change this with every update in the model file

Reply:
After downloading, Renamed to a generic DCGAN_fashionGen.pth